util/src/DigestImpl.cpp

```diff
@@ -37,56 +38,67 @@ using namespace CryptoPP;
 
 namespace concord::util::digest {
 
-size_t DigestUtil::digestLength() { return DigestType::DIGESTSIZE; }
+////////////////////////////////////////////
+// CryptoppDigestCreator implementations.
+////////////////////////////////////////////
+CryptoppDigestCreator::CryptoppDigestCreator() {
```

Comment:
The CryptoppDigestCreator should be in cryptopp_digest_creator.cpp file. Also the Open ssl creator should be in a different .cpp file.

Reply:
We can just rename this file.

Reply:
Addressed in https://github.com/arc-vmware/concord-bft/pull/3/commits/b7fc8a50c49f6f986000ec33fefe4d8e1580802d